public/css/icinga/forms.less

```diff
@@ -329,6 +332,16 @@ form.icinga-form {
   button[type="button"] {
     line-height: normal;
   }
+
+  // Button elements of type button inside a form controls
+  .form-controls button[type="button"] {
+    .button();
+  }
+
+  .control-group button[type="button"]:hover {
+    color: @text-color-on-icinga-blue;
+    background-color: @icinga-blue;
+  }
```

Comment:
You don't need these two blocks. Since you've added the `button[type=button]` above where `.btn-remove` styles are applied, the hover effect is applied correctly and the style anyway. Cancel buttons shouldn't look like standard buttons, they don't change anything after all. Remove this.